traitsui/qt4/ui_panel.py

```diff
@@ -257,7 +257,6 @@ def panel(ui):
         panel = QtGui.QTabWidget()
         # Identify ourselves as being a Tabbed group so we can later
         # distinguish this from other QTabWidgets.
-        panel.setProperty("traits_tabbed_group", True)
```

Comment:
What's the purpose of this change - it seems unrelated (similarly with https://github.com/enthought/traitsui/pull/276/files#diff-a749063742d071c56800949d439502a2L554 below).